nemo/collections/asr/jasper.py

```diff
@@ -44,6 +44,21 @@ class JasperEncoder(TrainableNM):
                     'tied' (bool)  # Whether to use the same weights for all
                         # sub-blocks.
                         # Defaults to False
+                    'se' (bool)  # Whether to add Squeeze and Excitation
+                        # sub-blocks.
+                        # Defaults to False
+                    'se_reduction_ratio' (int)  # The reduction ratio of the Squeeze
+                        # sub-module.
+                        # Must be an integer > 1.
+                        # Defaults to 16
+                    'kernel_width' (float)  # Conv kernel size multiplier
```

Comment:
Maybe it is worth to rename it to something like 'kernel_size_factor', 'kernel_width_factor', 'width_factor'? In my experience, 'kernel_size' and 'kernel_width' are synonyms for 1D convolution. Here it is definitely a factor to scale the kernel size.

Reply:
Changed to `kernel_size_factor` to remain in line with `kernel_size` naming convention.